### ISSUE TITLE
Why is noise outside the san plot

### ISSUE THREAD

**Issue by lpatiny** [cheminfo/nmrium#1098]
Is it normal than for negative points the noise is outside the san plot curve ?

![image](https://user-images.githubusercontent.com/1484241/118656962-4f443600-b7eb-11eb-9343-e769a9c615d3.png)

**Comment by lpatiny** [comment]
@djeanner What do you think ?

**Comment by jobo322** [comment]
we take 5 (factorStd) times the noise level, It is possible the reason. Now, If you prefer we could return both values

**Comment by djeanner** [comment]
Yes, the line is the SNR, only signal should be above. In the negative channel nothing goes beyond the noise level.
